download.html

```diff
@@ -0,0 +1,32 @@
+---
+layout: page
+title: "Download Terasology Launcher"
+header-img: "/img/ss_3.jpg"
+---
+<div class="row ">
+  <div class="col s6 l2 center">
+    <h4>Select OS:</h4>
+  </div>
+  <div class="col s6 l7 center">
+    <select style="display: block;" id="OS">
+      <option>Windows (64-bit)</option>
+      <option>Windows (32-bit)</option>
+      <option>MacOs</option>
```

Comment:
Please use the correct spelling "macOS" (https://de.wikipedia.org/wiki/MacOS).
```suggestion
      <option>macOS</option>
```